src/main/java/bio/terra/service/filedata/flight/ingest/IngestPopulateFileStateFromFileGcpStep.java

```diff
@@ -42,10 +46,16 @@ public StepResult doStep(FlightContext context) {
     FlightMap inputParameters = context.getInputParameters();
     BulkLoadRequestModel loadRequest =
         inputParameters.get(JobMapKeys.REQUEST.getKeyName(), BulkLoadRequestModel.class);
-    GoogleBucketResource bucketResource =
-        FlightUtils.getContextValue(context, FileMapKeys.BUCKET_INFO, GoogleBucketResource.class);
-    Storage storage = gcsPdao.storageForBucket(bucketResource);
```

Comment:
It looks like this was the only use of `gcsPdao.storageForBucket()`. If so, the method can be remove from `GcsPdao`.

Reply:
Nice catch!

Reply:
Actually, there were a few usages of this within other `GcsPdao` methods.

Reply:
Whoops, I guess I only looked outside of the class.